adds commands for working with images in docker

audibleblink opened the pull request:
Added one of my most used commands starting to troubleshoot why a process isn't starting in a container.

felixonmars commented:
I even wanted to add `docker exec -it {{container}} bash` too :P

ostera commented on pages/common/docker.md:
```diff
@@ -18,3 +18,11 @@
+
+`docker images`
+
+- Get a shell inside of an image
```
Not correct, this would execute the bash command within a new container. The more likely useful command is to get a shell within an already running container, `docker exec -it <name> /bin/bash`

Unfortunately this commands' semantics are quite similar.

felixonmars replied:
Actually it is correct since he didn't mention container at all, maybe a wording issue. Changing "inside of" to "with" may work better :)

(I also mentioned `docker exec` though :P)

ostera commented on pages/common/docker.md:
```diff
@@ -18,3 +18,11 @@
 
 `docker stop {{container}}`
+
+- List all images
```
Not very useful since it's right there in the command's help. A useful tldr would be `docker images --filter="dangling=true"` for all untagged images, or `docker images -qa` for all images ids (ideal for a cleanup).

audibleblink replied:
I agree, but I guess I'm unclear if this utility is supposed outline the most used commands or provide helpful tips. I used previous commands here as reference, like `docker ps`, which is also in the command's help.

I just took it off to keep the list shorter

ostera commented:
:-1:

audibleblink commented:
I agree with `docker images` not being as helpful as seeing some actual use-cases, but I guess I'm unclear if this utility is supposed outline the most used commands or provide helpful tips. I used previous commands here as reference, like `docker ps`, which is also in the command's help.

I just took it off to keep the list shorter and corrected the terms for `docker run` and added `docker exec`

denis-sokolov commented:
`docker run -it` and `docker exec

igorshubovych commented:
LGTM

felixonmars commented:
:+1:

Now up to @ostera :)
